src/test/java/io/meeds/qa/ui/steps/definition/ManageSpaceStepDefinitions.java

```diff
@@ -328,9 +329,9 @@ public void rejectRandomSpace(String spacePrefix) {
     homeSteps.rejectRandomSpaceInvitation(randomSpaceName);
   }
 
-  @Given("^Remove application button is displayed$")
-  public void removeAppButtonIsDisplayed() {
-    manageSpaceSteps.removeAppButtonIsDisplayed();
+  @Given("^Remove option of '(.*)' application is displayed$")
```

Comment:
You can combine three conditions in single one IMO by making the button label parameterized

```suggestion
  @Given("^(.*) option of the application '(.*)' is displayed$")
```

Reply:
Done